integration/containerd/cri/integration-tests.sh

```diff
@@ -60,10 +74,22 @@ ci_cleanup() {
 	if [ "$ID" == ubuntu ] &&  [ -n "${CI}" ] ;then
 		[ -f "${kata_config}" ] && sudo rm "${kata_config}"
 	fi
+
+	if [ -n "${FACTORY_TEST}" ]; then
+		echo "destroy vm template"
+	        sudo -E PATH=$PATH "$RUNTIME" factory destroy
+	fi
+
```

Comment:
Should this function "reset" the changes made in `ci_config()`? In other words, undo:

- `enable_template = true`
- `internetworking_model = "bridged"`

Reply:
Hi @jodh-intel 
The ci_cleanup() will "reset" the changes by  remove the config file.

Reply:
Great! ;)